src/api/status/public/js/build-log/api.js

```diff
@@ -1,5 +1,5 @@
 // http://localhost:1111/pages/build.html -> http://localhost:4000/:path
```

Comment:
Can we change the comment also so we reflect the change and not confuse future readers.